flink-examples/doc/flink-high-error-count-alert/filters/01-file-input.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
+input {
+    file {
+        path => "/tmp/access.log"
```

Comment:
Is this path still relevant since the Dockerfile step suggests `-v /tmp/access.log:/opt/data/access.log \`?

Reply:
yes, the -v would map /tmp/access.log on the host to /opt/data/access.log in the container, so user can adding lines to /tmp/access.log on the host instead of manipulating files inside container.